SUMMARY.md

```diff
@@ -122,14 +122,21 @@
   - [Deploying C2D](infrastructure/compute-to-data-minikube.md)
   - [C2D - Private Docker Registry](infrastructure/compute-to-data-docker-registry.md)
 - [🤑 DeFi](defi/README.md)
-- [💰 Data Farming Rewards](rewards/README.md)
-  - [Farming 101](rewards/df-intro.md)
-  - [veOCEAN](rewards/df-veocean.md)
-  - [Curate with Volume DF](rewards/df-volumedf.md)
-  - [Predict with Challenge DF](rewards/df-challengedf.md)
-  - [Complete User Guides](rewards/df-guides.md)
-  - [Emissions & APYs](rewards/df-emissions-apys.md)
-- [🤝 Contribute](contribute/README.md)
-  - [Partners & Collaborators](contribute/projects-using-ocean.md)
-  - [Contributor Code of Conduct](contribute/code-of-conduct.md)
-  - [Legal Requirements](contribute/legal-reqs.md)
+* [👀 Predictoor](predictoor/README.md)
+  * [Learn about Predictoor](predictoor/pdr-intro.md)
+  * [Architecture](predictoor/pdr-architecture.md)
+  * [Parameters](predictoor/pdr-parameters.md)  
+  * [How to Earn](predictoor/pdr-earn.md)  
+  * [Predictoor FAQ](predictoor/pdr-faq.md)
+* [💰 Earn](rewards/README.md)
```

Comment:
Recame `Earn` -> `Data Farming Rewards`

Why:
- There are many ways to earn: DF, as a predictoor, as a trader, and Data Challenges. So we need to be precise here
- And to flip it around: if I'm in docs and searching for "Data Farming" or "Rewards" by looking at the sidebar TOC, I can't easily find it. This fixes it

Reply:
Done. 

Tried changing it to "Earn" to KISS and match the rest of the docs' spine.
I would have also enjoyed seeing veOCEAN in there somewhere.